content/docs/command-reference/import-url.md

```diff
@@ -200,11 +200,11 @@ produces a regular stage in `dvc.yaml`.
   from the source. The default value is `4 * cpu_count()`. Using more jobs may
   speed up the operation.
 
-- `--version-aware` - capture cloud versioning information when importing the
-  file. By default, DVC will automatically capture cloud versioning information
-  if the URL contains a cloud versioning ID. When `--version-aware` is provided
-  along with a URL that does not contain a cloud versioning ID, DVC will capture
-  the latest version of the file.
+- `--version-aware` - capture [cloud versioning] information of the current
+  version when importing the file. DVC will always
```

Comment:
is it different for version aware and not version aware? 

(sorry, I'm still a bit confused about the semantics I guess)

Reply:
Yes. Without `--version-aware`, data will be pushed to the remote and pulled from there if available (although now we also try to fall back to the source if we can). With `--version-aware`, no data is pushed to the remote and data is always pulled from the source.

Reply:
Let me know if you think we can clarify it better. It's also described in similar language higher in the page -- there is a part of the description that explains how cloud versioning works with import-url.